README.md

```diff
@@ -19,26 +19,39 @@
 
 It is meant to bridge the gap between the different uses of your algorithms. With `modelkit` you can ensure that the same exact code will run in production, on your machine, or on data processing pipelines.
 
-## Features
+## Quickstart
 
-`modelkit`'s key features are:
+`modelkit` provides a straightforward and consistent way to wrap your prediction  code in a `Model` class:
```

Comment:
We might want to chain the key features you talk about down below, and why not get inspired by the Requests Readme (which I love): https://docs.python-requests.org/en/master/

I thought of this, but it might be better up above:
> `modelkit` is a minimalist yet powerful MLOps library for Python, built for people who want to deploy ML models to production. It packs several key features which make your go-to-production journey a breeze:
// CODE EXAMPLE